package.json

```diff
@@ -10,14 +10,17 @@
     "test": "standard && tap test.js"
   },
   "author": "David Mark Clements",
+  "contributors": ["Johannes Fahrenkrug"],
```

Comment:
None of the other contributor/mainainers to this module are listed, so I don't think it would be fair about adding you here.

Reply:
@mcollina Sure, I didn't know that. I'll remove it.